fix: improve sd-expandable a11y (#1724)
solid-design-system/solid: .changeset/violet-bulldogs-draw.md

---
'@solid-design-system/components': patch
'@solid-design-system/docs': patch
---

Improve sd-expandable spacings according to design.
Improve sd-expandable a11y:

> - Reorder of sd-expandable dom order. (button first so it is targetable by keyboard first)
> - Improvement of the sd-expandable template labels
> - Added accessibility hints to sd-expandable

Reviewer: ```suggestion
- Reorder elements in the DOM (button first so it is targetable by keyboard on first tab)
- Improved template labels
- Added accessibility hints
```
